changes/ce/fix-11337.en.md

```diff
@@ -0,0 +1 @@
+Fix return error when topic rewrite targets a topic with wildcards. Now it returns error 400 (Bad Match) instead of erro 500 (Internal Error).
```

Comment:
```suggestion
Fix HTTP API error when a publish topic rewrite rule targets a topic with wildcards. Now it returns error 400 (Bad Match) instead of error 500 (Internal Error).
```

Reply:
Ack